src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Defaults.cs

```diff
@@ -30,4 +30,10 @@ public static class Defaults
     /// in the <see cref="IResponseCacheProvider"/>'s cache before they are considered expired and evicted.
     /// </summary>
     public static TimeSpan DefaultTimeToLiveForCacheEntries { get; } = TimeSpan.FromDays(14);
+
+    /// <summary>
+    /// Defines the version number for the reporting format. If and when the serialized format undergoes breaking changes, this number
+    /// will be incremented to indicate the version a report file was created with compared to the version this library will produce.
+    /// </summary>
+    public const int ReportingFormatVersion = 1;
```

Comment:
What's the expected use of this public const?

Reply:
Good catch - I think this should be internal.

Reply:
Ah but I guess it has to be public so that it can be used as the default value in the constructor 🤔

Reply:
The thought was that someone using the library might want to check to determine which version of the reporting format is supported, but I'm not opposed to making it internal.